examples/twobee/setup.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+import io
+import re
+
+from setuptools import setup, find_packages
+
+with io.open('./twobee/__init__.py', encoding='utf8') as version_file:
+    version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]", version_file.read(), re.M)
+    if version_match:
+        version = version_match.group(1)
+    else:
+        raise RuntimeError("Unable to find version string.")
+
+
+with io.open('README.rst', encoding='utf8') as readme:
+    long_description = readme.read()
+
+
+setup(
+    name='TwoBee',
+    version=version,
+    description='Simple game demo',
+    long_description=long_description,
+    author='BeeWare Project',
+    author_email='contact@beeware.org',
+    license='BSD license',
+    packages=find_packages(
+        exclude=[
+            'docs', 'tests',
+            'windows', 'macOS', 'linux',
+            'iOS', 'android',
+            'django'
+        ]
+    ),
+    python_requires='>=3.5',
+    package_data={
+        'twobee': ['resources/*'],
+    },
+    install_package_data=True,
+    classifiers=[
+        'Development Status :: 1 - Planning',
+        'License :: OSI Approved :: BSD license',
+    ],
+    install_requires=[
+    ],
+    options={
+        'app': {
+            'formal_name': 'TwoBee',
+            'bundle': 'org.beeware.widgets'
+        },
+
+        # Desktop/laptop deployments
+        'macos': {
+            'app_requires': [
+                'toga-cocoa',
```

Comment:
The packaging problems you're be seeing will be because this won't be installing the dev versions by default; you need to explicitly pin a .dev version in the requirements for it to be installed.

Reply:
I’m sure you’ve thought about this but if platform updates keep preventing things to build over time then perhaps somekind of built in warning when its trying to build on an OS it wasn’t designed to build on? Otherwise the user experience in these transition times could be poor.